modules/uhttp/internal/stats/metrics.go

```diff
@@ -33,31 +33,48 @@ const (
 	TagMiddleware = "middleware"
 )
 
-// HTTPTags creates metrics scope with defined tags
 var (
-	HTTPTags = map[string]string{
+	httpTags = map[string]string{
 		TagModule: "http",
 		TagType:   "request",
 	}
-
-	// HTTPPanicCounter counts panics occurred in http
-	HTTPPanicCounter tally.Counter
-	// HTTPAuthFailCounter counts auth failures
-	HTTPAuthFailCounter tally.Counter
-	// HTTPMethodTimer is a turnaround time for http methods
-	HTTPMethodTimer tally.Scope
-	// HTTPStatusCountScope is a scope for http status
-	HTTPStatusCountScope tally.Scope
 )
 
-// SetupHTTPMetrics allocates counters for necessary setup
-func SetupHTTPMetrics(scope tally.Scope) {
-	httpScope := scope.Tagged(HTTPTags)
-	HTTPPanicCounter = httpScope.Counter("panic")
+// Client is a client for stats.
```

Comment:
Nit: I think metrics were moved to internal package, because of the globals, now they can become private structs in the uhttp package.